src/FffGcodeWriter.cpp

```diff
@@ -1602,8 +1602,113 @@ void FffGcodeWriter::processSpiralizedWall(const SliceDataStorage& storage, Laye
     const bool is_top_layer = ((size_t)layer_nr == (storage.spiralize_wall_outlines.size() - 1) || storage.spiralize_wall_outlines[layer_nr + 1] == nullptr);
     ConstPolygonRef wall_outline = part.insets[0][0]; // current layer outer wall outline
     const int seam_vertex_idx = storage.spiralize_seam_vertex_indices[layer_nr]; // use pre-computed seam vertex index for current layer
+
+    std::vector<float> flows(wall_outline.size(), 1.0); // flow multiplier for each line segment
+    std::vector<Point> shifts(wall_outline.size()); // per vertex shift to compensate for line width changes
+
+    const coord_t default_line_width = mesh.settings.get<coord_t>("wall_line_width_0");
+    const double min_line_width = mesh.settings.get<Ratio>("spiralize_min_line_width");
+    const double max_line_width = mesh.settings.get<Ratio>("spiralize_max_line_width");
+
+    if (min_line_width != 1.0 || max_line_width != 1.0)
+    {
+        // wall line width can vary
+
+        auto get_line_width = [&](const Point& prev_point, const Point& this_point, const Point& next_point, coord_t& line_width, Point& bisector, double& abs_sine)
+        {
+            const double corner_rads = LinearAlg2D::getAngleLeft(prev_point, this_point, next_point);
+
+            bisector = rotate(normal(next_point - this_point, default_line_width * 5), corner_rads / 2);
+            abs_sine = std::abs(std::sin(corner_rads / 2));
+
+            Polygons lines;
+            lines.addLine(this_point - bisector, this_point + bisector);
+#if 0
```

Comment:
Debug code; should probably be removed.

Reply:
It's very useful to have that but as it offends you I will remove it.